shuffler/auxiliary/SW Sea/Mercay Island/Mercay.json

```diff
@@ -0,0 +1,316 @@
+          "contents": "quiver_upgrade",
+          "overlay": -1,
+          "overlay_offset": "TODO"
+        },
+        {
+          "name": "RandomTreasure",
+          "type": "island_shop",
+          "contents": "common_treasure",
+          "overlay": 31,
+          "overlay_offset": "0x2f54"
+        }
+      ],
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "Tavern",
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "TreasureSelling",
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "MercayHouse",
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "Shipyard",
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "RockHouse",
+      "doors": [
+        {
+          "name": "Outside",
+          "link": "MercayTown.Main"
+        }
+      ]
+    },
+    {
+      "name": "AboveMercayTown",
+      "doors": [
+        {
+          "name": "South",
+          "link": "MercayTown.Main"
+        },
+        {
+          "name": "West",
+          "link": "PathToToTOK.Main"
+        },
+        {
+          "name": "ToBridgeCave",
+          "link": "BridgeCave.Main"
+        },
+        {
+          "name": "Grotto",
+          "link": "GrottoToLedge.Down"
+        },
+        {
+          "name": "SouthLedge",
+          "link": "MercayTown.Ledge"
+        },
+        {
+          "name": "IslandGrotto",
+          "link": "BridgeCave.Main"
+        }
+      ],
+      "chests": [
+        {
+          "name": "RollIntoTree",
+          "type": "tree",
+          "contents": "big_green_rupee",
+          "zmb_file_path": "Map/isle_main/map02.bin/zmb/isle_main_02.zmb",
+          "zmb_mapobject_index": 25
+        },
+        {
+          "name": "AboveEntrance",
+          "type": "chest",
+          "contents": "wisdom_gem",
+          "zmb_file_path": "Map/isle_main/map02.bin/zmb/isle_main_02.zmb",
+          "zmb_mapobject_index": 54
+        }
+      ]
+    },
+    {
+      "name": "PathToToTOK",
+      "doors": [
+        {
+          "name": "East",
+          "link": "AboveMercayTown.Main"
+        },
+        {
+          "name": "ToTOK",
+          "link": "TODO - link to Temple of Ocean King area"
+        },
+        {
+          "name": "Grotto",
+          "link": "GrottoToLedge.Up"
+        }
+      ]
+    },
+    {
+      "name": "GrottoToLedge",
+      "chests": [
+        {
+          "name": "PowerGem",
+          "type": "chest",
+          "contents": "power_gem",
+          "zmb_file_path": "TODO",
+          "zmb_mapobject_index": -1
```

Comment:
Strangely, this area doesn't seem to be a part of the `isle_main` map. I searched every zmb file, and none of them have a chest containing a power gem. 🤷

Reply:
You may check the `dngn_first` map since it seems to hold some data about grottos/caves on Mercay iirc.

Reply:
I checked, unfortunately they're not there either. It looks like `dngn_first` only contains the two rooms for the cave that you go through at the beginning of the game to bypass the broken bridge